src/ORM/DataList.php

```diff
@@ -1361,15 +1366,14 @@ private function fetchEagerLoadManyMany(
         $joinRows = [];
         if (!empty($parentIDs) && !empty($fetchedIDs)) {
             $fetchedIDsAsString = implode(',', $fetchedIDs);
-            $joinRows = DB::query(
-                'SELECT * FROM "' . $joinTable
-                // Only get joins relevant for the parent list
-                . '" WHERE "' . $parentIDField . '" IN (' . implode(',', $parentIDs) . ')'
-                // Exclude any children that got filtered out
-                . ' AND ' . $childIDField . ' IN (' . $fetchedIDsAsString . ')'
-                // Respect sort order of fetched items
-                . ' ORDER BY FIELD(' . $childIDField . ', ' . $fetchedIDsAsString . ')'
-            );
+            $joinRows = SQLSelect::create()
+                ->setSelect('"' . $joinTable . '".' . "*")
+                ->setFrom([$joinTable => $joinTable])
```

Comment:
```suggestion
                ->setFrom('"' . $joinTable . '"')
```
There's no need to set an alias or use an array, but we _do_ need to escape the table name.